### DISCUSSION TITLE
Help to run Zingg on Databricks for Customer 360

### DISCUSSION THREAD

**Discussion by iitbhumanish** [zinggAI/zingg#568] [Q&A] [answered]
I have tried to run Zingg Identity Resolution on Databricks for Customer 360 and followed the steps written below link. I get stuck in labelling step and seeking help to move into  right direction  would be appreciated.

https://www.zingg.ai/documentation-article/identity-resolution-on-databricks-for-customer-360
1.	Run Spark – submit job with own custom config file   it ran fine and created unmarked data at below location.
 
![image](https://user-images.githubusercontent.com/131442577/233591832-f4705cd5-2e0f-49e9-9e01-c3bfe7cbd08d.png)

2.	run notebook zinggLabeler.py  to label   it shows two  pair records ( not sure why it shows two, I believe it should show only one at a time)  but does not proceed further after labelling through  the widget . 
https://github.com/zinggAI/zingg/blob/main/examples/databricks/zinggLabeler.py
![image](https://user-images.githubusercontent.com/131442577/233591953-ec40b236-02e8-49dc-90eb-88a93ecfe548.png)

Just to test , I answered 2 and page is refreshed but did not show new record rather it shows same record.
![image](https://user-images.githubusercontent.com/131442577/233591997-fb2c7368-89cb-4e87-82a5-38d1d779d3de.png)

**Comment by sonalgoyal** [comment]
Seems like you moved to the newer notebook, so closing this one.

**Reply by iitbhumanish** [reply]
please do not close it . This issue is related to Spark Submit job so any suggestion here would be helpful.  I am discussion other issue ( running zingg through python script not through Spark Submit job)  on other notebook

**Comment by sonalgoyal** [comment, marked as answer]
We are not supporting the older spark job version
